libs/core/langchain_core/prompts/chat.py

```diff
@@ -407,21 +407,26 @@ def from_template(
         cls: Type[_StringImageMessagePromptTemplateT],
         template: Union[str, List[Union[str, _TextTemplateParam, _ImageTemplateParam]]],
         template_format: str = "f-string",
+        *,
+        partial_variables: Optional[Dict[str, Any]] = None,
         **kwargs: Any,
     ) -> _StringImageMessagePromptTemplateT:
         """Create a class from a string template.
 
         Args:
             template: a template.
             template_format: format of the template.
+            partial_variables: A dictionary of variables that can be used too partially.
             **kwargs: keyword arguments to pass to the constructor.
 
         Returns:
             A new instance of this class.
         """
         if isinstance(template, str):
             prompt: Union[StringPromptTemplate, List] = PromptTemplate.from_template(
-                template, template_format=template_format
+                template,
+                template_format=template_format,
+                partial_variables=partial_variables,
             )
             return cls(prompt=prompt, **kwargs)
         elif isinstance(template, list):
```

Comment:
if you pass in a list of templates and partial variables, they will be silently ignored. imo we should not just silently ignore. we can (a) raise a nice error saying this isnt supported, (b) try to only pass relevant things to each template, (c) something else?

i think id prefer (a)? should also test this codepath more, whatever we decide

Reply:
> if you pass in a list of templates and partial variables, they will be silently ignored. imo we should not just silently ignore. we can (a) raise a nice error saying this isnt supported, (b) try to only pass relevant things to each template, (c) something else?
> 
> i think id prefer (a)? should also test this codepath more, whatever we decide

Sorry for the slow reply, I have solved it.